HomeAssistant/Classes/RegionManager.swift

```diff
@@ -264,7 +264,7 @@ extension RegionManager: CLLocationManagerDelegate {
 
         let locationAge = Current.date().timeIntervalSince(last.timestamp)
         if locationAge > kLocationMaximumAge {
-            print("Location is older than threshhold. ")
+            print("Location is older than threshhold.")
```

Comment:
Can you fix the spelling of "threshold" too please?
```suggestion
            print("Location is older than threshold.")
```